spacy/lang/sl/lex_attrs.py

```diff
@@ -0,0 +1,116 @@
+from ...attrs import LIKE_NUM
+from ...attrs import IS_CURRENCY
+import unicodedata
+
+
+_num_words = set(
```

Comment:
I get your point about performance, but I feel at least the inflected forms of the numbers up to 10 should be included here, e.g. `dveh` `trem` `petimi`.

Reply:
After looking at the inflicted forms, basically few different forms cover all the inflicted forms of the masculine, feminine and neutral cases so I have just included all of them. I have also updated the inflicted forms of the currencies.

Reply:
I have tried to note down all the inflicted forms and it turns out that a lot of them overlap. So everything combined isn't all that much. That should pretty much be everything. I should have looked at it sooner. :)